cmd/run.go

```diff
@@ -232,8 +287,21 @@ func createAggSender(
 	go blockNotifier.Start(ctx)
 	log.Infof("Starting epochNotifier: %s", epochNotifier.String())
 	go epochNotifier.Start(ctx)
-	return aggsender.New(ctx, logger, cfg, agglayerClient,
+	aggsender, err := aggsender.New(ctx, logger, cfg, agglayerClient,
 		l1InfoTreeSync, l2Syncer, epochNotifier, l1EthClient, l2Client, rollupDataQuerier)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create AggSender: %w", err)
+	}
+	// TODO: Remove this, just for testing
+	if cfg.Mode == "PessimisticProof" {
+		validator, err := createAggSenderValidator(ctx, cfg, l1InfoTreeSync,
+			l2Syncer, l1EthClient, l2Client, rollupDataQuerier)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create AggSender validator: %w", err)
+		}
+		aggsender.AttatchValidator(validator)
```

Comment:
Just curious, how we are going to add a new validator to the aggsender? Is this just a temporary function, but in future when we fully integrate, we are going to dial the aggsender validator using the grpc and marshal the request?